src/main.js

```diff
@@ -39,7 +72,7 @@ fullCastbutton.addEventListener("click", () => {
       });
   }
 
-  //This is th function that creates the card structure
+  //This is the function that creates the card structure
   function createCard(character) {
     const characterSection = document.createElement("section");
     if (status === "fullCast") {
```

Comment:
Hi Adriana,
instead of adding the class related to the input status, you could also add the class depending on the info given from the api: `character.status === dead` for example. then you should also have the background when the filter is not active. :)